app/src/main/java/org/gdg/frisbee/android/eventseries/TaggedEventSeriesFragment.java

```diff
@@ -108,6 +108,7 @@ EventAdapter createEventAdapter() {
         return new EventAdapter(getActivity(), mTaggedEventSeries.getDefaultIconResId());
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
These warnings are dangerous actually and we have to have something to fix that. These caused crashes in the past. 
Can you remove them? :smile: